### PR TITLE
TMA-616: Provisioning Brick: no implicit conversion of Symbol into In…

### PR DESCRIPTION
**Pull request by vhtien**
…teger

### CODE REVIEW

File: lib/gooddata/lcm/actions/provision_clients.rb [hunk 1]
```diff
@@ -67,7 +67,8 @@ def call(params)
           rescue => e
             params.gdc_logger.error "Problem occurs when provisioning clients. Purge all invalid clients now ..."
             res = PurgeClients.send(:call, params)
-            deleted_client_ids = res.select { |r| r[:status] == 'purged' }.map { |r| r[:client_id] }
+            params.gdc_logger.debug "Purge clients result: #{res}"
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/provision_clients.rb]: Calls 'params.gdc_logger' 3 times (DuplicateMethodCall)

Comment (yenkins) [inline lib/gooddata/lcm/actions/provision_clients.rb]: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)

File: lib/gooddata/lcm/actions/provision_clients.rb [hunk 2]
```diff
@@ -67,7 +67,8 @@ def call(params)
...
+            deleted_client_ids = res[:results].select { |r| r[:status] == 'purged' }.map { |r| r[:client_id] }
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/provision_clients.rb]: Has the variable name 'r' (UncommunicativeVariableName)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/352/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/85/ : SUCCESS in 29s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/395/ : SUCCESS in 36s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/139/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/93/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/403/ : SUCCESS in 1m 46s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1000/ : FAILURE in 16m 37s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/759/ : UNSTABLE in 4m 10s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/143/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/97/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/407/ : SUCCESS in 1m 46s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1004/ : FAILURE in 3m 45s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/763/ : SUCCESS in 22m 10s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/144/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/99/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/409/ : SUCCESS in 29s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1006/ : FAILURE in 25m 57s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/764/ : SUCCESS in 21m 22s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/149/ : FAILURE in 11s
- gooddata-ruby-pronto-code-review-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED
- gooddata-ruby-integration-tests-zuul : SKIPPED
- gooddata-ruby-lcm-integration-tests-zuul : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/373/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/109/ : SUCCESS in 31s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/419/ : SUCCESS in 29s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/150/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/110/ : SUCCESS in 32s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/420/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1019/ : SUCCESS in 45m 45s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/777/ : SUCCESS in 25m 42s

**Pull request merged**